### DISCUSSION TITLE
bluetooth connect when switching to bluetooth device profile

### DISCUSSION THREAD

**Discussion by ghost** [Belphemur/SoundSwitch#1010] [FAQ]
Hey,
Can we have bluetooth connect when switching to bluetooth device profile? so it tries to connect to the device if it is not connected?

**Comment by Belphemur** [comment]
Hey,
I see the idea and that you've also created a issue enhancement for it :) 
I'll close this for now and rely on the issue.

**Reply by Belphemur** [reply]
https://github.com/Belphemur/SoundSwitch/issues/1008
